test/builder/QuarkBuilderCometSupply.t.sol

```diff
@@ -150,6 +150,95 @@ contract QuarkBuilderCometSupplyTest is Test, QuarkBuilderTest {
         assertNotEq(result.eip712Data.hashStruct, hex"", "non-empty hashStruct");
     }
 
+    function testSimpleCometSupplyMax() public {
+        QuarkBuilder builder = new QuarkBuilder();
+        Accounts.ChainAccounts[] memory chainAccountsList = new Accounts.ChainAccounts[](3);
+        chainAccountsList[0] = Accounts.ChainAccounts({
+            chainId: 1,
+            quarkStates: quarkStates_(address(0xa11ce), 12),
+            assetPositionsList: assetPositionsList_(1, address(0xa11ce), uint256(3e6)),
+            cometPositions: emptyCometPositions_()
+        });
+        chainAccountsList[1] = Accounts.ChainAccounts({
+            chainId: 8453,
+            quarkStates: quarkStates_(address(0xb0b), 2),
+            assetPositionsList: assetPositionsList_(8453, address(0xb0b), uint256(0)),
+            cometPositions: emptyCometPositions_()
+        });
+        chainAccountsList[2] = Accounts.ChainAccounts({
+            chainId: 7777,
+            quarkStates: quarkStates_(address(0xc0b), 5),
+            assetPositionsList: assetPositionsList_(7777, address(0xc0b), uint256(0)),
+            cometPositions: emptyCometPositions_()
+        });
```

Comment:
You don't need all this right?

Reply:
Let me try to remove those,  but I recalled if the list is not match with some other inputs, some error occurs. (forgot what error it got hit),  so I kept it as close as the one we've been used in setup function, but modify it to be 0 in other chain.)

Reply:
ok looks like this one worked out well haha.  Will removed other acount balnce.